instrumentation/servlet/servlet-common/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/servlet/v5_0/dispatcher/RequestDispatcherInstrumentation.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.javaagent.instrumentation.servlet.v5_0.dispatcher;
```

Comment:
Why this module has package name `v5_0`? It should be `common`, right?

Reply:
Indeed, somehow broke it with renaming in IntelliJ. Fixed.